modules/mysql/mysql.go

```diff
@@ -79,16 +79,19 @@ func StartContainer(ctx context.Context, opts ...MySQLContainerOption) (*MySQLCo
 	return &MySQLContainer{container, username, password, database}, nil
 }
 
-func (c *MySQLContainer) Username() string {
-	return c.username
-}
+func (c *MySQLContainer) ConnectionString(ctx context.Context) (string, error) {
```

Comment:
Question: do we want to allow consumers to customize the the connection string, probably adding parameters to it? Please see https://golang.testcontainers.org/modules/postgres/#connectionstring

Reply:
done!